framework/wazuh/core/common.py

```diff
@@ -53,6 +56,49 @@ def find_wazuh_path():
     return wazuh_path
 
 
+def call_wazuh_control(option) -> str:
+    wazuh_control = os.path.join(find_wazuh_path(), "bin", "wazuh-control")
+    try:
+        proc = subprocess.Popen([wazuh_control, option], stdout=subprocess.PIPE)
+        (stdout, stderr) = proc.communicate()
+        return stdout.decode()
+    except:
+        return None
+
+
+def get_wazuh_info(field) -> str:
+    wazuh_info = call_wazuh_control("info")
+    if not wazuh_info:
+        return "ERROR"
+
+    if not field:
+        return wazuh_info
+
+    env_variables = wazuh_info.rsplit("\n")
+    env_variables.remove("")
+    wazuh_env_vars = dict()
+    for env_variable in env_variables:
+        key, value = env_variable.split("=")
+        wazuh_env_vars[key] = value.replace("\"", "")
+
+    return wazuh_env_vars[field]
+
+
+@lru_cache(maxsize=None)
+def get_wazuh_version() -> str:
+    return get_wazuh_info("WAZUH_VERSION")
+
+
+@lru_cache(maxsize=None)
+def get_wazuh_revision() -> str:
+    return get_wazuh_info("TEST_REVISION")
+
+
+@lru_cache(maxsize=None)
+def get_wazuh_type() -> str:
+    return get_wazuh_info("TEST_TYPE")
```

Comment:
The `wazuh-control` tool returns **WAZUH_TYPE**, not `TEST_TYPE`.

Reply:
Fixed